### PR TITLE
[release] merging crate-release-update into gh-pages

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

File: atk/auto/range.rs.html [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <span class="doccomment">///</span>
     <span class="doccomment">/// # Implements</span>
     <span class="doccomment">///</span>
-    <span class="doccomment">/// [`RangeExt`](trait.RangeExt.html), [`WidgetExt`](trait.WidgetExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`BuildableExt`](trait.BuildableExt.html), [`OrientableExt`](trait.OrientableExt.html)</span>
+    <span class="doccomment">/// [`RangeExt`](trait.RangeExt.html), [`WidgetExt`](trait.WidgetExt.html), [`glib::object::ObjectExt`](../glib/object/trait.ObjectExt.html), [`BuildableExt`](trait.BuildableExt.html), [`OrientableExt`](trait.OrientableExt.html), [`WidgetExtManual`](prelude/trait.WidgetExtManual.html), [`BuildableExtManual`](prelude/trait.BuildableExtManual.html)</span>
```
Comment (EPashkin) [inline atk/auto/range.rs.html:77]: This sure  part only for gtkrange.

Comment (GuillaumeGomez) [reply atk/auto/range.rs.html:77]: What do you mean?

Comment (EPashkin) [reply atk/auto/range.rs.html:77]: `<record name="Range" c:type="AtkRange" glib:type-name="AtkRange" glib:get-type="atk_range_get_type" c:symbol-prefix="range">`
don't implement any of these interfaces.

Comment (GuillaumeGomez) [reply atk/auto/range.rs.html:77]: Then this is an old issue considering that it just added the missing manual ones. Do you have an idea of the origin of this bug? Maybe it's specific to records?

Comment (EPashkin) [reply atk/auto/range.rs.html:77]: Also no this text in atk/vendor.md

**Pull request closed without merging**